tests/ft_sh.vader

```diff
@@ -1,14 +1,58 @@
 Include: include/setup.vader
 
 Execute (shellcheck):
-  AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc']
-  set ft=sh
-  AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc', '-s', 'sh']
-  set ft=bash
-  AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc', '-s', 'bash']
-  set ft=nadda
-  AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc']
+  " By variable...
+  let g:is_sh = 1
+  AssertEqual ['-fgcc', '-s', 'sh'], neomake#makers#ft#sh#shellcheck().args
+  unlet g:is_sh
```

Comment:
You could use Vader's `Save`/`Restore` here, but it's only relevant to not mess state when running Vader inside of Vim I'd say.